### ISSUE TITLE
NavigationBar has 2 required props, WHY?

### ISSUE THREAD

**Issue by alinz** [Kureev/react-native-navbar#35]
Thanks for this extension.
Why did you make `navigator` and `route` props required? How would you set these arguments? None of the examples are setting these values. :(

**Comment by Kureev** [comment]
@alinz You can see how I set `navigator` and `route` in both examples I have. For example you can check [this lines](https://github.com/Kureev/react-native-navbar/blob/master/examples/basic/index.ios.js#L46-L51) from `basic` example and [this lines](https://github.com/Kureev/react-native-navbar/blob/master/examples/customElements/index.ios.js#L109-L111) from `customElements` example.

**Comment by alinz** [comment]
@Kureev, thanks for fast response. My question was based on [this](https://github.com/Kureev/react-native-navbar/blob/master/examples/customElements/index.ios.js#L128-L131). `NavigationBar` is not setting those two variables and react was showing warnings. Is there any way we can get ride of those warnings.

p.s. even with those warnings, I still like your extension. :+1:

**Comment by dubert** [comment]
@Kureev I get these warnings: 

```
Warning: Failed propType: Required prop `navigator` was not specified in `NavigationBar`. Check the render method of `CustomElements`.
index.ios.bundle:39889 Warning: Failed propType: Required prop `route` was not specified in `NavigationBar`. Check the render method of `CustomElements`.
```
when running the CustomElements example

**Comment by Kureev** [comment]
Fixed in `0.6.2`
